CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [2.1.2] - 2020-05-10
+### Update
+- Configurator API
```

Comment:
It's more a requirement update isn't it ?

Reply:
Not really, there is a new param in configuration API

Reply:
Oh yes sorry I didn't saw the previous commit on 0.4.0 :+1: